File: medplum-java/medplum-model/src/main/java/com/fhir/ImplementationGuide_Parameter.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableImplementationGuide_Parameter.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface ImplementationGuide_Parameter {
  Optional<List<Extension>> extension();

  Optional<String> id();

  Optional<List<Extension>> modifierExtension();

  Optional<Implementationguide_parameterCode> code();

  Optional<String> value();

  static ImmutableImplementationGuide_Parameter.Builder builder() {
    return ImmutableImplementationGuide_Parameter.builder();
  }
}

File: medplum-typescript/packages/sdk/src/model/fhir/ImplementationGuide_Parameter.ts
import { Extension } from "./Extension";
import { Implementationguide_parameterCode } from "./Implementationguide_parameterCode";

export interface ImplementationGuide_Parameter {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    code: Implementationguide_parameterCode | null | undefined;
    value: string | null | undefined;
}
